sessions/react-usestate-and-events-exercise-01/src/App.js

```diff
@@ -0,0 +1,21 @@
+/*
+Given are two buttons, the first is going to add
+and second one to substract 1.
+1. Write two functions, add() and subtract().
+Log something to the console.
+2. Add your add() and subtract() function
+to the relevant button.
+*/
+
```

Comment:
I don't understand the task. Should there be a useState that keeps track over the value? Or shoud these functions only log a small text? If its the latter, we should make this more explicit in the text.

Reply:
At this point, we did not speak about useState, so the answer is no. I thus think that our students don't need this explicit information since they don't know anything else.

The following third task of this exercise makes it clear, I think:
"As soon as you spoke about useState, implement a count state and save the changes done by your add() and subtract() function to it. Make sure that the count rendered in your App to be visible in the browser."

I still don't know what to do about this PR or whether we should wait to Dominik to update the useState sessions.